service/src/cli.yml

```diff
@@ -26,20 +26,44 @@ args:
         default_value: "9944"
     - ws-external:
         long: ws-external
-        help: let worker api listen to external requests too      
+        help: Set this flag in case the worker should listen to external requests.
     - mu-ra-port:
         short: r
         long: mu-ra-port
         help: Set the websocket port to listen for mu-ra requests
         takes_value: true
         default_value: "3443"
-    - worker-rpc-port:
+    - trusted-worker-port:
         short: P
-        long: worker-rpc-port
-        help: Set the websocket port where the worker rpc direct invocations port listens
+        long: trusted-worker-port
+        help: Set the trusted websocket port of the worker, running directly in the enclave.
         takes_value: true
         default_value: "2000"
-    
+    - untrusted-worker-port:
+        short: U
+        long: untrusted-worker-port
+        help: Set the untrusted websocket port of the worker
+        takes_value: true
+        default_value: "2001"
+    - trusted-external-address:
+        short: trusted-ext
```

Comment:
shouldn't we still offer a one letter short option (like U, p and P) which does not have to be verbose?

Reply:
In case you have to set external  addresses, you will probably be setting all three some them. I imagine it quite messy with one letter shortcuts, but I'm open for suggestions.

Reply:
I did not see that before, but AFAIK now, must not be more than one letter. This will not work.

Reply:
the idea is that you can still use the long or the short option, as you prefer.

Reply:
I see. I'll adapt then, thanks for pointing that out.

Reply:
Updated